docker/openemr/flex-edge/utilities/devtools

```diff
@@ -66,6 +66,18 @@ if [ "$1" == "php-parserror" ] || [ "$1" == "clean-sweep" ]; then
     find . -type f \( -name "*.php" -or -name "*.inc" \) \( -not -path "./vendor/*" -and -not -path "./node_modules/*" -and -not -path "./ccdaservice/node_modules}/*" \) -exec php -d error_reporting=32767 -l {} \; 2>&1 >&- | grep "^"
 fi
 
+if [ "$1" == "rector-dry-run" ]; then
+    echo "Rector PHP process dry-run changes"
+    cd /var/www/localhost/htdocs/openemr
+    php vendor/bin/rector process --dry-run
+fi
+
+if [ "$1" == "rector-process" ]; then
+    echo "Rector PHP process changes"
+    cd /var/www/localhost/htdocs/openemr
+    php vendor/bin/rector process
+fi
+
```

Comment:
```suggestion
if [ "$1" = rector-dry-run ]; then
    echo 'Rector PHP process dry-run changes'
    cd /var/www/localhost/htdocs/openemr
    php vendor/bin/rector process --dry-run
fi

if [ "$1" = rector-process ]; then
    echo 'Rector PHP process changes'
    cd /var/www/localhost/htdocs/openemr
    php vendor/bin/rector process
fi

```

Use `=` instead of `==` in test commands. The only place in shell where you'd use `==` is in arithmetic context.